portal/config/eproms/ScheduledJob.json

```diff
@@ -5,13 +5,27 @@
       "args": null,
       "id": 6,
       "kwargs": {
+        "research_study_id": 0,
         "org_id": 20000
       },
       "name": "IRONMAN site summary email",
       "resourceType": "ScheduledJob",
       "schedule": "r 20 * * 0",
       "task": "send_questionnaire_summary"
     },
+    {
+      "active": true,
+      "args": null,
+      "id": 6,
+      "kwargs": {
+        "research_study_id": 1,
+        "org_id": 20000
+      },
+      "name": "EMPRO site summary email",
+      "resourceType": "ScheduledJob",
+      "schedule": "r 20 20 * *",
```

Comment:
[At 20:rr on day-of-month 20](https://crontab.guru/#1_20_20_*_*)

ie, random minute, at 8pm on the 20th of every month :+1: